frontend/main.go

```diff
@@ -78,10 +90,31 @@ func main() {
 		config.Listen = *bindaddr
 	}
 
+	if *splaytreeenabled {
+		config.SplayTreeEnabled = *splaytreeenabled
+	}
+
+	if *lz4enabled {
+		config.LZ4Enabled = *lz4enabled
+	}
+
+	if *aolenabled {
+		config.AOLEnabled = *aolenabled
+	}
+
 	http.HandleFunc("/", handler)
 	defer unload()
 	var httperr error
 	log.Println("Starting server...")
+	if config.LZ4Enabled {
+		log.Println("LZ4 compression is enabled")
+	}
+	if config.AOLEnabled {
+		log.Println("Append-only log is enabled")
+	}
+	if config.SplayTreeEnabled {
+		log.Println("Splay-tree log is enabled")

```

Comment:
No I mean the text should be "Splay-tree is enabled". The splay tree isn't a log.